packages/protocol/contracts/tokenvault/BridgedERC20.sol

```diff
@@ -70,19 +79,64 @@ contract BridgedERC20 is
         srcDecimals = _decimals;
     }
 
+    function startOutboundMigration(address to)
```

Comment:
We do not have `stopOutboundMigration()` -> But i guess that is fine-> most prob. not necessary, if we close the `stopInboundMigration` in the new contract.